src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -302,6 +306,17 @@ private async Task RestoreAsync(bool forceRestore, RestoreOperationSource restor
                 .GroupBy(x => x.ProjectStyle)
                 .ToDictionary(x => x.Key, y => y.Count());
 
+            bool areNamespacesEnabled = _packageNamespacesConfiguration?.AreNamespacesEnabled ?? false;
+            int numberOfSourcesWithNamespaces = 0;
```

Comment:
Given that we cut modes, do you think the number of soruces with namespaces is helpful?

I'm kind of leaning toward removing them.

Reply:
Sure, I removed it. Also if number of sources with namespaces then number of entries packageNamespaces doesn't make much sense, so I'm removing it.